manual.md

```diff
@@ -2,13 +2,14 @@
 * [Ingest Data](#Ingest-data)
 * [Create apass and sloan catalogs](#Create-apass-and-sloan-catalogs-for-new-objects)
 * [Running the pipeline](#Running-the-pipeline)
+* [Cookbook](#Cookbook)
 * [Creating a Landolt Catalog](#Creating-a-Landolt-Catalog)
 * [Difference Imaging](#Difference-Imaging)
 * [Definitions](#Definitions)
 
 # Ingest data
-* Download observations from SNEx and run ingesttar.py
-* Download observations from archive.lco.global using LCOGTingest.py. This will only work for public data (e.g., standard stars images and observations from over 1 year ago) unless you have an account for the observing portal. For example (all arguments optional; see ```LCOGTingest.py -h``` for options):
+* Download observations from SNEx and run ingesttar.py. This data will have already been reduced with BANZAI.
```

Comment:
Will this data also have been reduced by snex at all? Is this for floyds data?

Reply:
This is only photometry and my understanding was that the images you download as a tar file from SNEx is reduced